src/lamp_py/runtime_utils/remote_files.py

```diff
@@ -37,6 +37,14 @@ class RemoteFileLocations:
         bucket_name=springboard_bucket,
         file_prefix=os.path.join(tm_prefix, "STOP_CROSSING"),
     )
+    tm_daily_work_piece = S3Location(
```

Comment:
add more S3Locations.

i feel like this class is ok for now, but could see a world where we want to break it up into TransitMasterLocations, GtfsRtLocations, RailPerformanceLocations, BusPerformanceLocations, etc.